### PR TITLE
[UPDATE] - Refactor error log on app/import.go

### PR DESCRIPTION
**Pull request by andikanugraha11**

### CODE REVIEW

File: app/import.go [hunk 1]
```diff
@@ -19,7 +18,7 @@ import (
 
 func stopOnError(err LineImportWorkerError) bool {
 	if err.Error.Id == "api.file.upload_file.large_image.app_error" {
-		mlog.Warn(fmt.Sprintf("Large image import error: %s", err.Error.Error()))
+		mlog.Warm("Large image import error", mlog.Err(err))
```
Comment (jespino) [inline app/import.go]: this is failing because the real error is `err.Error`, can you fix that?

Comment (streamer45) [reply app/import.go]: Also there's a typo. It should be `mlog.Warn`.

Comment (streamer45) [reply app/import.go]: Also there's a typo. It should be `mlog.Warn`.

**Comment by hanzei** [comment]
@andikanugraha11 Do you have any questions about the provided feedback?

**Comment by hanzei** [comment]
Please note that we will close this PR after another 10 days of inactivity. If you don't object, other contributors might complete this PR for you.

**Comment by hanzei** [comment]
Im closing this PR due to inactivity. Still, thanks for the PR @andikanugraha11

**Pull request closed without merging**